### DISCUSSION TITLE
Avoid Directives hydration warning

### DISCUSSION THREAD

**Discussion by silencerspirit** [nuxt/nuxt#16039] [Questions]
Hi!

I have a `loading.ts` file with the following code:

![code](https://user-images.githubusercontent.com/22055696/207858856-43c7bc82-9932-404e-8d05-1a816477f2f1.png)

How i can avoid warning about hydration when use `v-loading.modifier="(boolean)"` - ?

**Comment by silencerspirit** [comment]
Still, I thought that with this approach (showing the loader through a directive) is not an option, need to do it through a component (abstraction).

But maybe I'm wrong and someone will give me an idea? 😊

**Comment by TheAlexLichter** [comment]
1. Please do not post your code as image
2. You can work around that problem by defining the directive on SSR as `{}` (use `process.server` to check)
